src/ensembl/src/content/app/species-selector/components/species-selector-app-bar/SpeciesSelectorAppBar.tsx

```diff
@@ -48,6 +48,8 @@ export const SpeciesSelectorAppBar = (props: Props) => {
 };
 
 const SelectedSpeciesList = (props: Props) => {
+  const shouldLinkToGenomeBrowser =
+    props.selectedSpecies.filter(({ isEnabled }) => isEnabled).length > 0;
```

Comment:
@azangru we may have to disable the launchbar button based on selected enabled species count. Do you think it can be done in this PR?

Reply:
Yes.

Reply:
Cool, are you okay to make that change?

Reply:
Yep. Done.